### ISSUE TITLE
Use python3-libselinux instead of libselinux-python on RHEL8/CentOS8

### ISSUE THREAD

**Issue by rtsisyk** [cloudalchemy/ansible-prometheus#244]
```
TASK [cloudalchemy.prometheus : Install SELinux dependencies] ************************************************************************
FAILED - RETRYING: Install SELinux dependencies (5 retries left).
FAILED - RETRYING: Install SELinux dependencies (4 retries left).
FAILED - RETRYING: Install SELinux dependencies (3 retries left).
FAILED - RETRYING: Install SELinux dependencies (2 retries left).
FAILED - RETRYING: Install SELinux dependencies (1 retries left).
failed: [infra.hel.diceplex.io] (item=libselinux-python) => {"ansible_loop_var": "item", "attempts": 5, "changed": false, "failures": ["No package libselinux-python available."], "item": "libselinux-python", "msg": "Failed to install some of the specified packages", "rc": 1, "results": []}
FAILED - RETRYING: Install SELinux dependencies (5 retries left).
FAILED - RETRYING: Install SELinux dependencies (4 retries left).
^C [ERROR]: User interrupted execution
```

```
dnf install python-libselinux
Failed to set locale, defaulting to C
Last metadata expiration check: 0:11:04 ago on Sun Nov 17 06:25:40 2019.
No match for argument: python-libselinux
Error: Unable to find a match
```

```
dnf info python3-libselinux
Failed to set locale, defaulting to C
Last metadata expiration check: 0:11:19 ago on Sun Nov 17 06:25:40 2019.
Installed Packages
Name         : python3-libselinux
Version      : 2.8
Release      : 6.el8
Arch         : x86_64
Size         : 616 k
Source       : libselinux-2.8-6.el8.src.rpm
Repo         : @System
From repo    : BaseOS
Summary      : SELinux python 3 bindings for libselinux
URL          : https://github.com/SELinuxProject/selinux/wiki
License      : Public Domain
Description  : The libselinux-python3 package contains python 3 bindings for developing
             : SELinux applications.
```

**Comment by paulfantom** [comment]
Which version of the role are you using?

**Comment by paulfantom** [comment]
Fixed in #248
